src/CUITe/Controls/UITesting/ApplicationUnderTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections.Generic;
+using CUITe.SearchConfigurations;
+using CUITControls = Microsoft.VisualStudio.TestTools.UITesting.WinControls;
+
+namespace CUITe.Controls.UITesting
+{
+    /// <summary>
+    /// Application under test
+    /// </summary>
+    /// <seealso cref="CUITe.Controls.ControlBase{Microsoft.VisualStudio.TestTools.UITesting.ApplicationUnderTest}" />
+    public class ApplicationUnderTest : ControlBase<Microsoft.VisualStudio.TestTools.UITesting.ApplicationUnderTest>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ApplicationUnderTest"/> class.
+        /// </summary>
+        /// <param name="searchConfiguration">The search configuration.</param>
+        public ApplicationUnderTest(By searchConfiguration = null)
+            : this(new Microsoft.VisualStudio.TestTools.UITesting.ApplicationUnderTest(), searchConfiguration)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ApplicationUnderTest"/> class.
+        /// </summary>
+        /// <param name="sourceControl">The source control.</param>
+        /// <param name="searchConfiguration">The search configuration.</param>
+        public ApplicationUnderTest(Microsoft.VisualStudio.TestTools.UITesting.ApplicationUnderTest sourceControl, By searchConfiguration = null)
+            : base(sourceControl, searchConfiguration)
+        {
+        }
+
+        /// <summary>
+        /// Gets the parent of the control.
+        /// </summary>
+        /// <exception cref="System.NotImplementedException"></exception>
+        public override ControlBase Parent
+        {
+            get { throw new System.NotImplementedException(); }
```

Comment:
Is it correct to throw `NotImplementedException`, or should you throw `NotSupportedException` or similar?